.idea/dictionaries/kevint.xml

```diff
@@ -0,0 +1,7 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="kevint">
+    <words>
+      <w>terasology</w>
+    </words>
+  </dictionary>
+</component>
```

Comment:
It is your dictionare.

Reply:
It is. I was worried too. But apparently this is the way it's supposed to work. See the section about dictionaries on https://keturn.net/blog/posts/2020/05/git-add-dot-idea-share-intellij/

(oh, I should add `id` fragments to those sections)